src/WindowManager.vala

```diff
@@ -1290,12 +1308,11 @@ namespace Gala
 				unminimize_completed (actor);
 			if (end_animation (ref minimizing, actor))
 				minimize_completed (actor);
-			if (end_animation (ref maximizing, actor))
-				size_change_completed (actor);
-			if (end_animation (ref unmaximizing, actor))
-				size_change_completed (actor);
```

Comment:
Because that signal is already being called here: https://github.com/elementary/gala/blob/7e6444a8ae0ca26ab37a08fb51e79ce2471e1ced/src/WindowManager.vala#L780

It appears that at least with the version of mutter in Loki, `size_change_completed` has to be called right away after executing the method of `size_change`, if not it crashes

Rico and I both tried to run that method on the `transitions_completed` signal connections on both maximize and unmax, but it crashes on Loki (although not on Juno)